balances.json

```diff
@@ -0,0 +1,8 @@
+{
+    "db_username": "idseqmaster",
```

Comment:
i don't see where this is referenced, also is it okay for this to be exposed publicly?

Reply:
Removed; snuck in from earlier drafts. People would still need a password anyways for this to be useful, and that didn't make it in here.